tests/test_rooms/test_room_analytics.py

```diff
@@ -115,12 +113,6 @@ def test_analytics_for_monthly_bookings(self):
             get_bookings_count_monthly_response
         )
 
-    def test_analytics_for_bookings_invalid_date_range(self):
-        CommonTestCases.admin_token_assert_equal(
-            self, get_bookings_count_invalid_date_range,
-            get_bookings_count_invalid_date_range_response
-        )
-
     def test_analytics_for_monthly_bookings_diff_years(self):
```

Comment:
This test was removed because it was testing the invalid dates but with the current implementation there are no invalid dates.